packages/runtime/src/modules/DeciderModule.ts

```diff
@@ -63,13 +62,11 @@ export class DeciderModule extends RuntimeModule {
                 }
 
                 if (result === "ManualRequestDecisionRequired") {
-                    if (!requestId) throw new Error("Request ID is required for manual decision required result.");
-
                     this.runtime.eventBus.publish(
                         new RelationshipTemplateProcessedEvent(event.eventTargetAddress, {
                             template,
                             result: result as RelationshipTemplateProcessedResult.ManualRequestDecisionRequired,
-                            requestId
+                            requestId: request.id
```

Comment:
undo, and re-add the above

Reply:
But why?

Reply:
and you can check that the request is valid

Reply:
Can you please elaborate on how the Request's validity was checked before in contrast to now?

Reply:
From my point of view this only checks if the function is used correctly. With my change, there no longer is the risk to use it incorrectly. Also, I think readability actually improves, since an unnecessary parameter is removed.

Reply:
yeah 👯